vyked/registry.py

```diff
@@ -216,7 +216,17 @@ def receive(self, packet: dict, protocol, transport):
         elif request_type == 'pong':
             self._ping(packet)
         elif request_type == 'ping':
-            self._pong(packet, protocol)
+            if 'payload' in packet:
```

Comment:
What is this logic trying to do? How about a comment explaining the motive? Better still move this to a private method that has a docstring indicating the purpose of this.